tests/integration/backup/one_to_one.go/utils.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2022 Democratized Data Foundation
```

Comment:
todo: Fix copyright year.

```suggestion
// Copyright 2023 Democratized Data Foundation
```